release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -515,7 +515,6 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 |Supported
 |====
 
-[id="ocp-release-notes-insights-operator-enhancements_{context}"]
```

Comment:
Looks like this anchor was deleted inadvertently? Should add this back before merge